xacro/substitution_args.py

```diff
@@ -138,6 +137,7 @@ def _dirname(resolved, a, args, context):
 
 
 def _eval_find(pkg):
+    from ament_index_python.packages import get_package_share_directory, PackageNotFoundError
```

Comment:
`PackageNotFoundError` is not used:
```suggestion
    from ament_index_python.packages import get_package_share_directory
```